src/main/webapp/js/build-message-div.js

```diff
@@ -42,7 +42,71 @@ function formatTimestamp(messageTime){
           timestamp.getHours()+":"+timestamp.getMinutes()+":"+timestamp.getSeconds();
 }
 
-function buildMessageInTimeline(message, flip){
+function createDeleteButton(deleteConfirmationDiv){
+  var btn = document.createElement('button');
```

Comment:
nit: change variable to something a bit more descriptive, I wasn't sure if this was the delete button or cancel button at first.